<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>644</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="year_widget">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>37</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>17</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>250</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="year_hun">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>38</y>
     <width>34</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>년도 </string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>138</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="marks_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>간  단  요  약</string>
   </property>
  </widget>
  <widget class="QLabel" name="amount_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>헌금액</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hun_date">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>36</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>헌금일자 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="hun_name">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>72</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>기타소득 명칭 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="marks_widget">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>148</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="other_name_combo_widget">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>72</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>100</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>626</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user_widget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>4</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="add_other_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>헌금등록</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>173</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount_widget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>148</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="file_close_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>닫기(저장취소)</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="other_save_button">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hap_total">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>230</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>등록되는 헌금총액 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hap_total_widget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="other_date_widget">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>35</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDate|Qt::ImhPreferNumbers</set>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="month_hun">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>38</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>월</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_widget">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>37</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="week_hun">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>38</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>몇번째 주 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="week_widget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>37</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="Bankincome_check">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>38</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="locale">
    <locale language="Kinyarwanda" country="Rwanda"/>
   </property>
   <property name="text">
    <string>통장예입</string>
   </property>
   <property name="autoRepeatDelay">
    <number>302</number>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="other_income_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>581</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>통장예입</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="re_calculate_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>합계금액 재계산</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_row_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>선택행삭제</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>108</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="hun_name_2">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>74</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>회계구분</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_name_combo_widget">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>74</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>week_widget</tabstop>
  <tabstop>gubun_name_combo_widget</tabstop>
  <tabstop>other_name_combo_widget</tabstop>
  <tabstop>amount_widget</tabstop>
  <tabstop>marks_widget</tabstop>
  <tabstop>add_other_input</tabstop>
  <tabstop>year_widget</tabstop>
  <tabstop>month_widget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>add_other_input</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>other_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>re_calculate_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>re_calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_row_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>remove_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>other_save_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>file_close_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>660</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>file_save()</slot>
  <slot>re_calculate()</slot>
  <slot>file_close()</slot>
  <slot>remove_row()</slot>
  <slot>slot1()</slot>
  <slot>other_input()</slot>
 </slots>
</ui>
